src/Nordea/Components/Spinner.elm

```diff
@@ -0,0 +1,11 @@
+module Nordea.Components.Spinner exposing (..)
+
+import Html.Styled exposing (Attribute, Html, div)
+import Nordea.Resources.Icons as Icons
+
+
+view : List (Attribute msg) -> Html msg
+view attrs =
+    div
+        attrs
```

Comment:
Should we set Colors.blueDeep as the default color and a default size?

Reply:
The more (sane) defaults, the better imo! We might even specify sizes in s, m, l, etc. since it's probably not going to vary _that_ much.

Reply:
I agree blueDeep should be the default color yeah. I can create a default with the original size and a small with the 3.75rem size.